pycheribuild/projects/cross/opensbi.py

```diff
@@ -227,6 +227,25 @@ def setup(self):
         super().setup()
         self.make_args.set(FW_TEXT_START=0x80000000)
 
+    def install(self, **kwargs):
+        self.makedirs(self.install_dir)
+        for platform in self.all_platforms:
+            args = self.make_args.copy()
+            args.set(PLATFORM=platform)
+            self.run_make_install(cwd=self.source_dir, options=args)
+        # Install into the QEMU firmware directory so that `-bios default` works
+        qemu_fw_dir = BuildCheriAllianceQEMU.get_install_dir(self, cross_target=CompilationTargets.NATIVE) / "share/qemu/"
```

Comment:
Add a hook to the base class to get the QEMU project instead? That avoids this code duplication.